store/kv/fdb.go

```diff
@@ -416,6 +416,7 @@ func (t *ftx) Read(_ context.Context, table []byte, key Key) (baseIterator, erro
 		return nil, err
 	}
 
+	// the iterator should be change if we need to support reverse order because the keys may be chunked by the caller.
```

Comment:
Sorry this comment doesn't quite make sense.

Reply:
let me expand it, but basically we can't support reverse range by default as the chunker will be relying on some ordering to perform the merge. I have added a note here so that we don't forget to return a different iterator in that case.